staging/src/k8s.io/apiserver/pkg/server/options/recommended.go

```diff
@@ -57,9 +62,20 @@ func (o *RecommendedOptions) AddFlags(fs *pflag.FlagSet) {
 	o.Audit.AddFlags(fs)
 	o.Features.AddFlags(fs)
 	o.CoreAPI.AddFlags(fs)
+	o.Admission.AddFlags(fs)
 }
 
-func (o *RecommendedOptions) ApplyTo(config *server.RecommendedConfig) error {
+// ApplyTo adds RecommendedOptions to the server configuration.
+// scheme should not be nil.
```

Comment:
add some docs here: it's the scheme of the apiserver types that are sent to the admission chain.

Reply:
ok